habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BehaveBddTestMojo.java

```diff
@@ -74,6 +75,12 @@ public class BehaveBddTestMojo extends AbstractHabushuMojo {
     @Parameter(property = "habushu.disableOutputCapture", defaultValue = "true")
     protected boolean disableOutputCapture;
 
+    /**
+     * Environment variables utilized in the behave bdd tests.
+     */
+    @Parameter(property = "habushu.behaveTestEnvironmentVariables")
+    protected Map<String, String> behaveTestEnvironmentVariables = null;
+
```

Comment:
S: Should update the documentation with this new config and a description of it.